src/drivers/driver_zephyr.c

```diff
@@ -538,6 +538,9 @@ void wpa_drv_zep_event_proc_assoc_resp(struct zep_drv_if_ctx *if_ctx,
 		if (if_ctx->ft_roaming) {
```

Comment:
@GaofengZhangNXP please re-write the commit message with proper English, it is very difficult to understand what it is saying. Also please start the sentences with capital letter so that it is easier to separate the sentences.
So the commit message should say the reasoning why this change is being done, this is not clear from the message.

Reply:
@jukkar I have updated the commit, please help review it, thanks